aarch64/root/firmware-update.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 
 VERSION=0.5
+FLASHER=/usr/bin/GCFFlasher_internal
+FW_PATH=/usr/share/deCONZ/firmware/
+FW_BASE=http://deconz.dresden-elektronik.de/deconz-firmware/
 
```

Comment:
Adding this for the preview-10-latest fw's and prefill (read) feature
```suggestion
FW_ONLINE=\$(curl -s \$FW_BASE?C=M;O=D'|sed -n 's/.*href=\"\([^\"]*\).*/\1/p'|grep -m 10 -E '^deCONZ_.*_0x.+[a-f0-9].bin.GCF\$')
FW_LATEST=\$(echo \$FW_ONLINE|tr \" \" \"\n\"|grep -m 1 GCF)
```

Reply:
I would put the list into an array. In that case the "latest" would be the first element in the array:
```shell
FW_LOCAL=( $(ls -1 "$FW_PATH") )
FW_ONLINE=( $(curl -s "$FW_BASE?C=M;O=D" | grep -Po '(?<=href=")deCONZ_[^"]+_0x[0-9a-f]+\.bin\.GCF(?=")') )
FW_LATEST=$FW_ONLINE

echo "additional files online (10-latest)"
printf "    %s\n" "${FW_ONLINE[@]}" | grep -m 10 -v "$(printf "%s\n" "${FW_LOCAL[@]}")"
```

@marthoc what do you think about the additional output? I wouldn't add the default `FW_LATEST` to the read as it won't fit the used hardware on most installations ...